packages/web3-eth/types/index.d.ts

```diff
@@ -364,6 +364,9 @@ export class Eth {
         blockNumber: BlockNumber,
         callback?: (error: Error, result: GetProof) => void
     ): Promise<GetProof>;
+
+    requestAccounts(): Promise<string[]>
+    requestAccounts(callback: (error: Error, result: string[]) => void): Promise<string[]>
```

Comment:
In the types above here, a different style is used where callback is optional. Do you prefer having each format listed separately?

Reply:
As I'm informed is the overloaded way the preferred way because of the better hints TS devs do get. Should I update the whole file you commented on to have the consistency we would like to have?

Reply:
Oh I was just curious about this - that's good to know. I don't have any problem with mixed styles here - would probably leave as you have it just because changing all the other ones is a separate topic.